src/types/granule.graphql

```diff
@@ -172,6 +178,8 @@ type Query {
     options: JSON @deprecated(reason: "Use `params.options`")
     "Search for granules by orbit number min and max"
     orbitNumber: JSON @deprecated(reason: "Use `params.orbitNumber`")
+    "Search by passes which is part of the track information of a granule."
+    passes: JSON
```

Comment:
All of the parameters directly on the granule query are deprecated, these values should only be added to GranulesInput